content/courses/program-security/closing-accounts.md

```diff
@@ -270,7 +200,7 @@ participation in the lottery.
 ### 1. Setup
 
 Start by getting the code on the `starter` branch from the
-[following repo](https://github.com/Unboxed-Software/solana-closing-accounts/tree/starter).
+[following repo](https://github.com/solana-developers/solana-closing-accounts/tree/starter).
```

Comment:
The repo is at https://github.com/solana-developers/closing-accounts now. (ie, no extra `solana` in the name)